### DISCUSSION TITLE
New themes

### DISCUSSION THREAD

**Discussion by andeon** [tahoma2d/tahoma2d#729] [Show and tell]
I thought about creating compatible themes between Opentoonz and Tahoma2d. But I noticed that the default theme path is slightly different, maybe in the next versions the icons and style sheet will start to get more inconsistent*. That's it, isn't it?

 *Like these two examples:
>Tahoma2d - Status bar at the bottom (it's not at OTZ)
>Opentoonz - The slider to Zoom control in the Xshee (it's not at T2d) 

I edited a .qss file to create it, but found that this is not an efficient way, especially optimizing each new version with upstream changes or even to create some extra themes. ~~Is there any way to generate a custom theme, inheriting a default theme base, just changing the color set?~~ 
(I found a way using .less files with Visual Studio Code and the Easy LESS extension on the Linux, of course.)

![tahoma theme](https://user-images.githubusercontent.com/11843239/126013212-e7762891-204c-4831-ad08-56a9829d31f7.jpg)

My plan at this point is to create a repository for [opentoonz themes](https://github.com/andeon/opentoonz-themes) and port them to tahoma2d later.

**Comment by andeon** [comment]
All right, starting work!

https://github.com/andeon/tahoma2d-themes

![harlequin_t2d](https://user-images.githubusercontent.com/11843239/171545400-254a757f-6c3e-4da2-88ec-7ebc3ab958e2.jpg)

**Reply by andeon** [reply]
Six themes for Tahoma2d https://github.com/andeon/tahoma2d-themes

I'm gonna publish soon releases to keep the UI consistent with the stable versions.

**Comment by ghost** [comment]
Awesome! Thanks.

**Comment by ghost** [comment]
can you send me the openToonz theme in the T2D zip for OpenToonz? I
actually really like it

**Comment by andeon** [comment]
The idea was to have a unique Tahoma theme for Opentoonz and vice versa. Maybe in a future release I'll port this theme for now, trying to keep the UI consistent with updates to the next stable release.
